presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java

```diff
@@ -830,6 +830,11 @@ public void testStringFunctions()
 
         // Reverse
         assertQuery("SELECT comment, reverse(comment) FROM orders");
+
+        // Hamming_distance
+        assertQuery("SELECT hamming_distance('abcde', 'abcde')");
```

Comment:
Please make this a separate test method, ideally two, for easier debuggability when something fails.